rhc-ose-ansible/roles/post_install/tasks/main.yaml

```diff
@@ -0,0 +1,45 @@
+---
+
+  - name: "backup image streams"
+    command: cp /usr/share/ansible/openshift-ansible/roles/openshift_examples/files/examples/v1.1/image-streams/image-streams-rhel7.json /usr/share/ansible/openshift-ansible/roles/openshift_examples/files/examples/v1.1/image-streams/image-streams-rhel7.json.orig
+    when: target_registry is defined
+
+  - name: "Configure ImageStreams"
+    command: sed -i -e 's/registry.access.redhat.com/{{ target_registry }}/g' /usr/share/ansible/openshift-ansible/roles/openshift_examples/files/examples/v1.1/image-streams/image-streams-rhel7.json
+    when: target_registry is defined
```

Comment:
Any objections to submitting this upstream as a feature addition for the openshift_examples role?

I would suggest a variable openshift_examples_source_registry, and that the imagestream json file(s) be copied to a temporary location before doing the sed replace.